app/components/forms/wizard/basic-details-step.js

```diff
@@ -48,6 +48,10 @@ export default Component.extend(FormMixin, EventWizardMixin, {
     return this.get('settings.isOmiseActivated') && find(paymentCurrencies, ['code', this.get('data.event.paymentCurrency')]).omise;
   }),
 
+  canAcceptAliPay: computed('data.event.paymentCurrency', 'settings.isAliPayActivated', function() {
```

Comment:
@mrsaicharan1 Please add the corresponding field for `AliPay` in the Dictionary file too .

Reply:
Dictionary? Where do you mean?

Reply:
Thanks for pointing that out @kushthedude